lib/stddiff.ml

```diff
@@ -11,65 +11,71 @@ type 'a atomic_modification = { reference : 'a; current : 'a }
     [reference] is the value before and [current] is the value after the change
     occured. Use this type when there is no better representation available. *)
 
-type ('item, 'diff) map = {
-  same_map : 'item String_map.t;
-  changed_map : ('item, 'diff) entry String_map.t;
-}
-
-type ('same, 'change) maybe_changed = Same of 'same | Changed of 'change
 type 'item atomic_entry = ('item, 'item atomic_modification) entry
+type ('same, 'change) maybe_changed = Same of 'same | Changed of 'change
 
-type 'same maybe_changed_atomic =
-  ('same, 'same atomic_modification) maybe_changed
+module List_ = struct
```

Comment:
I think we can remove the trailing `_` here. `Stddif` shouldn't be globally open so it won't shadow stdlib's modules.